homeworks/pavel.krasovskiy_everett-young/homework_3/homework3.js

```diff
@@ -0,0 +1,20 @@
+var height = 4;
```

Comment:
please use const instead of var

Reply:
Done! Thanks.

Reply:
YAW, It seems, there are more constants in your code ;)

Reply:
fixed)